torchao/csrc/cuda/nms.cu

```diff
@@ -0,0 +1,181 @@
+#include <ATen/ATen.h>
```

Comment:
ok i guess as a first useful kernel to merge, I'm gonna adapt this example to use paged attention